packages/plugin-core/src/web/extension.ts

```diff
@@ -43,6 +44,17 @@ async function setupCommands(context: vscode.ExtensionContext) {
         await cmd.run();
       })
     );
+
+  if (!existingCommands.includes(CopyNoteURLCmd.key)) {
+    context.subscriptions.push(
+      vscode.commands.registerCommand(
+        CopyNoteURLCmd.key,
+        async (_args: any) => {
+          await container.resolve(CopyNoteURLCmd).run();
```

Comment:
we should add a test whenever we add a new resolve to make sure that the container is sufficiently configured to be able to instantiate this object.  It should be a two-liner to `setupWebExtContainer.test.ts`: see

```typescript
        const cmd = container.resolve(NoteLookupCmd);

        assert(!_.isUndefined(cmd));
```